### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Logs_Keyboard: Focus is lost after navigating from ‘Logs’ tab item

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3595]
**GitHub Tags:** #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.4.3; 

[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated:** 2.4.3

**Repro Steps:**
1)Launch IntelliJ IDEA
2)Tab till Create New Project button and press enter. Navigate to Azure Spark/HDInsight-->Spark Project Scala and press Next button and fill all the details to create a new project
3)Tab till Azure Spark/HDInsight-->Spark Project Scala
4)Tab till Build Tools-->Maven
5)Tab till Next button and press enter
6)Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
7)Tab till HDInsight and link a cluster
8)Navigate to project name-->src-->main-->scala and use keyboard shortcut key to open the context menu
9)Navigate to New-->Scala Class and press enter
10)Tab till object from menu and select it using keyboard
11)Override the code and Navigate to project name-->src-->main-->scala-->Project Name and right click on it & Tab till Submit Spark Application and press enter
12)Fill all the required details & Tab till finish button and press enter
13)Navigate to jobs under the cluster (which we have used) and press enter
14)Navigate to all the controls under logs tab & then press tab again
15) Verify if focus is lost after navigating from directory info under the Logs tab or not

**Actual Result: ​**
Focus is lost after navigating from directory info under the ‘Logs’ tab item

**Expected Result: ​**
Focus should not be lost after navigating from the ‘Logs’ tab

**User Impact: ​**
Users will find difficulty in navigating if focus is lost while navigating in the page

**MAS Reference: ​**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

Attachment:
[Bug_3595_focus is going to inviisble area.pptx](https://github.com/microsoft/azure-tools-for-java/files/3641835/Bug_3595_focus.is.going.to.inviisble.area.pptx)

**Comment by t-rufang** [comment]
Note: When the focus is at `Directory info` and we click tab, focus goes to somewhere invisible. When we click again, focus goes back to the link `Open Spark History UI`
![image](https://user-images.githubusercontent.com/32627233/65597774-7a519480-dfcc-11e9-9899-3bb5c396a174.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749698
